### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by EMK2000** [daktak/afh_downloader#9]
Sorry I cannot find other good place to give suggestion. 

I am really enjoy the downloader series, since it save my life to go to each website to download separate file. 

I just want to suggest the other downloader that maybe also useful: 
SuperSU, TWRP, CM, MoKee

**Comment by daktak** [comment]
Hi, thank you for your input.

I was of the opinion that CM would not be required as it has a built in OTA(?), so a user would not need to check and update manually .
I believe  https://play.google.com/store/apps/details?id=com.jmz.soft.twrpmanager provides this download feature for TWRP, is that not correct? I have not looked into SuperSU but thought that there was an existing app for updating/flashing this.

**Comment by EMK2000** [comment]
Just because I have SuperSU and Spider, and so the OTA does not work for me. I need to flash the full size every time. 

But never mind, the SuperSU is more important to me. As it is hard to find the correct link for different versions of stable and beta version of the flashable zip.

**Comment by EMK2000** [comment]
Spider → Xposed

**Comment by EMK2000** [comment]
Maybe also a APK Mirror downloader is also useful sometimes.

**Comment by EMK2000** [comment]
Yet another suggestion: 
Maybe the AFH downloader can support more than one flid at the same time. e.g. ROM + kernel... Like having different tabs.

**Comment by MarcRdC** [comment]
I agree: a bookmarking feature would be very useful.

**Comment by Ibuprophen** [comment]
@daktak, it would be great if a user was able to view a developers directory structure. 

For example, if I was to insert, add, bookmark, etc... the following (even without the "https://androidfilehost.com") from invisiblek in the app... 

https://androidfilehost.com/?w=profile&uid=9390034591667978340

... and be able to navigate the devices/directories within it. 

Just a suggestion... :-) 


~Ibuprophen

**Comment by daktak** [comment]
I agree, I was thinking maybe a tabbed interface, maybe comma delimit the flid field.
